crates/kind-openai/src/endpoints/chat.rs

```diff
@@ -199,6 +199,15 @@ impl<T> ChatCompletionChoice<T> {
 struct ChatCompletionResponseMessage<T> {
     content: T,
     refusal: Option<String>,
+    usage: Usage,
+}
+
+#[allow(dead_code)]
```

Comment:
Get rid of this, we're just going to make it public

Reply:
removed.